util.go

```diff
@@ -33,3 +35,20 @@ func (v VersionFlag) BeforeApply(app *Kong, vars Vars) error {
 	app.Exit(0)
 	return nil
 }
+
+// ChangeToFlag changes the current working directory to a path specified by a flag
+// early in the parsing process, changing how other flags resolve relative paths.
+//
+// Use this flag to provide a "git -C" like functionality.
+type ChangeToFlag string
+
+// Decode is used to create a side effect of changing the current working directory.
+func (c ChangeToFlag) Decode(ctx *DecodeContext) error {
```

Comment:
I think this would be better as an `AfterApply()` hook wouldn't it?

Reply:
I think it is too late then for `existingfile` to work (`existingfile` should resolve against changed directory).

Reply:
Aha! Makes sense 👍

Reply:
It is a big ugly but it works. :-) But I do not think there is any other more suitable and official hook.